data/build.gradle

```diff
@@ -6,8 +6,9 @@ targetCompatibility = 1.7
 dependencies {
     def dataDependencies = rootProject.ext.dataDependencies
     def dataTestDependencies = rootProject.ext.dataTestDependencies
+    def developmentDependencies = rootProject.ext.developmentDependencies
 
-    compile project(':domain')
+    implementation project(':domain')
```

Comment:
Please check if it can be built successfully since I had some issue on this (maybe)